File: 07. Perulangan Lanjut/B. Pola II/b_pola_ii.java
import java.util.Scanner;

class b_pola_ii {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int N = scan.nextInt();
        scan.close();
        for (int i = 0; i < N; i++){
            for (int j = 0; j < N; j++){
                if (j < N - i - 1){
                    System.out.print(" ");
                } else {
                    System.out.print("*");
                }
            }
            System.out.println();
        }
    }
}

File: 07. Perulangan Lanjut/B. Pola II/b_pola_ii.go
package main

import (
  "fmt"
)

func main() {
  var N int
  fmt.Scan(&N)
  for i := 0; i < N; i++ {
    for j := 0; j < N; j++ {
      if j < N-i-1 {
        fmt.Print(" ")
      } else {
        fmt.Print("*")
      }
    }
    fmt.Println()
  }
}
